src/api/vdom.js

```diff
@@ -321,6 +321,8 @@ export function element(tname, attrs, ...chren) {
       ch();
     } else if (ctype === 'string' || ctype === 'number') {
       newText(ch);
+    } else if (ctype === 'object') {
```

Comment:
`Array.isArray` is the right way IMHO, `'object'` can be `null`,`Object`, `Array`